core/v0/api/core.yaml

```diff
@@ -1694,7 +1695,7 @@ components:
           $ref: '#/components/schemas/Scalar'
     
     Document:
-      description: Describes a document which can be sent as a url
+      description: Describes a piece of written or printed matter in electronic form that provides information or evidence or that serves as an official record. This will be given a URL.<br>This has properties like url,label<br>This is used in the following situations.<br><ul><li>This is typically used in the order stage if any documents from the user are needed by the BPP to confirm the order</li><li>This is typically used in the fulfilment and post fulfillment stage if the BPP needs to send any documents related to the order to the user</li></ul><br>This is used to allow documents to be exchanged between the BAP and the BPP
```

Comment:
Suggest adding the following as well

`In the case of sensitive information being exchanged, such documents may be encrypted and additional authorization such as an an OTP or password may be implemented when the URL is accessed.`